datasets/eurlex/README.md

```diff
@@ -0,0 +1,232 @@
+  "eurovoc_concepts": ["192", "2356", "2560", "862", "863"]
+}
+```
+
+### Data Fields
+
+The following data fields are provided for documents (`train`, `dev`, `test`):
+
+`celex_id`: The official ID of the document. The CELEX number is the unique identifier for all publications in both Eur-Lex and CELLAR.\
+`document_type`: The type of the document (e.g., Directive, Regulation, Decision)\
+`title`: The title of the document.\
+`header`: The first part of the document, which includes the title and name of the legal body enforcing the legal act.\
+`recitals`: The second part of the document, which contains legal background references.\
+`main_body`: The main part of the document, organized in articles or even bigger units (e.g., chapters, sections).\
+`eurovoc_concepts`: The relevant EUROVOC concepts (labels).
```

Comment:
same suggestions as for the other dataset card:
- remove the \
- mention the fields that are actually available in the dataset loaded in python: celex_id, title, text, eurovoc_concepts
- mention the type of each field